lib/config.js

```diff
@@ -7,3 +7,6 @@ var ENABLE_PUSHSTATE = (IN_STATIC_SITE &&
 exports.IN_STATIC_SITE = IN_STATIC_SITE;
 exports.GENERATING_STATIC_SITE = GENERATING_STATIC_SITE;
 exports.ENABLE_PUSHSTATE = ENABLE_PUSHSTATE;
+
+// this is a dummy account until we choose a domain name
+exports.GA_ACCOUNT='UA-59356678-3';
```

Comment:
So as of #194, we're actually storing configuration in environment variables and transferring the whitelisted ones over to the client-side via webpack's `DefinePlugin`... #194 might be enough to show you how to integrate it into this PR, but if not, that's fine, we can just land this and I can integrate the `DefinePlugin` stuff in a separate PR.